README.md

```diff
@@ -1,58 +1,9 @@
-# Architecture
+# Elemental
 
-RancherOS v2 is an immutable Linux distribution built to run Rancher and
-it's corresponding Kubernetes distributions [RKE2](https://rke2.io) 
-and [k3s](https://k3s.io). It is built using the [cOS-toolkit](https://rancher-sandbox.github.io/cos-toolkit-docs/docs/)
-and based on openSUSE. Initial node configurations is done using a
-cloud-init style approach and all further maintenance is done using
-Kubernetes operators.
+Elemental is a software stack enabling a centralized, full cloud-native cluster management solution.
```

Comment:
```suggestion
Elemental is a software stack enabling a centralized, full cloud-native OS management solution with Kubernetes.
```

Reply:
Rancher Manager does cluster management, Elemental allows for OS management through Kubernetes 😉